classes/Routing/Controller/Controller.php

```diff
@@ -93,6 +94,7 @@ public function search()
             }
         }
         $resources = $this->addRelationshipsToSearch($resources, $this->request->get('strict'));
+        $resources = $this->addSortByToSearch($resources);
```

Comment:
Can we do something like below if we're re-using the same variable twice ?

`$resources = $this->addSortByToSearch($this->addRelationshipsToSearch($resources, $this->request->get('strict')));`

Reply:
Im not sure what you're asking

Reply:
Id say no as it makes it less clear. So far we have two lines, which add relaationships to the resource and then adds ordering to the resource. They can take place independently of each other and nesting them would make that less clear.